101-cosmosdb-create-arm-gremlin/README.md

```diff
@@ -0,0 +1,10 @@
+# Create a Gremlin Cosmos Database Account
```

Comment:
Please change to "Create an Azure Cosmos DB account for the Gremlin (Graph) API"